homeassistant/components/homekit/__init__.py

```diff
@@ -221,17 +213,7 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry):
     entity_config = options.get(CONF_ENTITY_CONFIG, {}).copy()
     auto_start = options.get(CONF_AUTO_START, DEFAULT_AUTO_START)
     safe_mode = options.get(CONF_SAFE_MODE, DEFAULT_SAFE_MODE)
-    entity_filter = convert_filter(
-        options.get(
-            CONF_FILTER,
-            {
-                CONF_INCLUDE_DOMAINS: [],
-                CONF_EXCLUDE_DOMAINS: [],
-                CONF_INCLUDE_ENTITIES: [],
-                CONF_EXCLUDE_ENTITIES: [],
-            },
-        )
-    )
+    entity_filter = FILTER_SCHEMA(options.get(CONF_FILTER, {}))
```

Comment:
```suggestion
    entity_filter = convert_filter(FILTER_SCHEMA(options.get(CONF_FILTER, {})))
```

We still need to convert it right?

Reply:
No not with `FILTER_SCHEMA`.  `FILTER_SCHEMA` is a `vol.All` of `BASE_FILTER_SCHEMA` + `convert_filter` so it just handles it.